shared/unattended/win7-32-autounattend.xml

```diff
@@ -169,53 +169,61 @@
 					<Order>5</Order>
 				</SynchronousCommand>
 				<SynchronousCommand wcm:action="add">
-					<CommandLine>%WINDIR%\System32\cmd /c net start telnet</CommandLine>
+					<CommandLine>%WINDIR%\System32\cmd /c tlntadmn config maxconn=30</CommandLine>
```

Comment:
is there any params that I check check in guest after instllation,
as I plan to update the num manually before, but didn't find the param